### PR TITLE
test: fix unstable test TestUpdateTs

### PR DESCRIPTION
**Pull request by ben1009**
### What problem does this PR solve? 
close https://github.com/pingcap/ticdc/issues/2345

### What is changed and how it works?
reduce the inner loop to speed up the test

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/puller/frontier/heap_test.go [hunk 1]
```diff
@@ -42,23 +42,30 @@ func (s *tsHeapSuite) TestUpdateTs(c *check.C) {
 	defer testleak.AfterTest(c)()
 	rand.Seed(0xdeadbeaf)
 	var heap fibonacciHeap
-	nodes := make([]*fibonacciHeapNode, 50000)
+	nodes := make([]*fibonacciHeapNode, 20)
```
Comment (amyangfei) [inline cdc/puller/frontier/heap_test.go:45]: maybe too small

**Review by amyangfei — COMMENTED** [review]
The test contains two part, insert N items to the heap and update heap item for N times. The insert phase is lightweight, and update phase may be cpu bound.
What about adding a timer to calculate how long is running in the test case itself (in the update heap phase), and exits loop before timeout.

**Comment by hi-rustin** [comment]
I've encountered this unstable test several times again. Could you please move forward as soon as possible? @ben1009  Thanks!  ❤️

**Comment by ben1009** [comment]
> I've encountered this unstable test several times again. Could you please move forward as soon as possible? @ben1009 Thanks! ❤️

❤️

**Review by hi-rustin — APPROVED** [review]
LGTM

**Review by amyangfei — APPROVED** [review]

**Pull request merged**